crates/miden-lib/asm/kernels/transaction/lib/output_note.masm

```diff
@@ -41,6 +41,24 @@ export.get_assets_info
     # => [note_data_ptr, num_assets]
 
     # get the assets commitment
-    exec.note::compute_output_note_assets_commitment
+    dup exec.note::compute_output_note_assets_commitment
+    # => [ASSETS_COMMITMENT, note_data_ptr, num_assets]
+
+    # get the start and the end pointers of the asset data
```

Comment:
nit: I would probably add a comment above this line to explain that the following code is inserting data into the advice map. Otherwise, I wasn't sure what this code does until I got to line 57 below.